app/views/posts/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<section class=pad>
+ <div class=border>
+    <div class="show-post">
+        <div class="user-name">Post by: <%=@post.author.name%></div>
+        <p class="comments-likes">Comments: <%=@post.comment_counter%>, Likes: <%=@post.likes_counter%></p>
+      </div>
+      <div class="show-detail">
+        <%=@post.text%>
+      </div>
+    </div>
+    <div class="comments">
+        <% @post.comments.each do |comment| %>
+          <%= render 'comment', comment: comment %>
+        <% end %>
+    </div>
+ </div>
+ </section>
```

Comment:
- [ ] Good job so far unfortunately I can't go to the post-show page. Please make sure to have a link takes to the post-show page and make sure that it displays the post with all comments listed as shown on  [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png) :+1: